vlm/src/main/scala/geotrellis/contrib/vlm/gdal/GDALBaseRasterSource.scala

```diff
@@ -27,11 +27,13 @@ import org.gdal.gdal.{Dataset, gdal}
 import org.gdal.osr.SpatialReference
 
 trait GDALBaseRasterSource extends RasterSource {
-  // options from previous transformation steps
+  /** -tap aligns the grid to the resolution, if the resolution is 5m, the cells are aligned to round multiples of 5. */
```

Comment:
What does `aligned` mean? What is their reference for the top left corner of the top-left pixel?